pages/clubInfoPage.js

```diff
@@ -0,0 +1,149 @@
+import Head from 'next/head'
+import clientPromise from '../lib/mongodb'
+import PhotoSlider from "../components/photoslider.js";
+{/* import NavBar from "../components/NavBar"*/}
+import { Button, Center, Box, Image, Flex, Badge, Text, ButtonGroup, Heading, Container, SimpleGrid, HStack, Avatar } from '@chakra-ui/react'
+const photos = ['photo1.jpg', 'photo2.jpg', 'photo3.jpg', 'photo4.jpg', 'photo5.jpg'];
+export const getServerSideProps = async () => {
+    try {
+      await clientPromise
+      return {
+        props: { isConnected: true },
+      }
+    } catch (e) {
+      console.error(e)
+      return {
+        props: { isConnected: false },
+      }
+    }
+  }
+export default function Home({
+    isConnected,
+  }) {
+return (
+  <div className="container">
+      <Head>
+        <title>GT Club Explorer Club Info Page</title>
+        <link rel="icon" href="/favicon.ico" />
+      </Head>
+
+      {/* <NavBar />*/}
+      <Box
+        display="flex"
+        justifyContent="center"
+        alignItems="center"
+        marginBottom="40px"
+      >
+        <Image
+          src="/img/loremIpsum.png"
+          width="100%"
+          height="100px"
+          objectFit="cover"
+        />
+      </Box>
+      <Container maxW='container.xl' mx="auto" spacing={16} marginBottom="40px">
+        <Flex alignItems="center">
+          <Avatar
+            size="2xl"
+            mx="auto"
+            display={{ base: "none", lg: "block" }}
+            borderRadius='full'
+            src="/img/LoremIpsumAboutPhoto.PNG"
+            marginRight="40px"
+          />
+          <Heading
+            fontSize={{ base: '5xl', lg: '7xl' }}
+            fontWeight='extrabold'
+          >
+            Georgia Tech Lorem Ipsum Association
+          </Heading>
+        </Flex>
+      </Container>
+      <Container maxW='container.lg' mx="auto" spacing={16}>
+        <Box backgroundColor= 'purple.100'
```

Comment:
for these, let's use the `Badge` Chakra UI component: https://chakra-ui.com/docs/components/badge

Reply:
fixed